src/openforms/forms/models/form.py

```diff
@@ -228,6 +228,18 @@ class Form(models.Model):
         ),
     )
     _is_deleted = models.BooleanField(default=False)
+    activate_on = models.DateTimeField(
+        _("activate on"),
+        blank=True,
+        null=True,
+        help_text=_("Date and time that the form should be activated."),
+    )
+    deactivate_on = models.DateTimeField(
+        _("deactivate on"),
+        blank=True,
+        null=True,
+        help_text=_("Date and time that the form should be deactivated."),
+    )
```

Comment:
I think that there should be some validation that these fields are in the future?

Reply:
Already discussed with Sergei (it was my first thought as well), but there are some use cases which would not be better if we had this validation.